src/pages/generate.tsx

```diff
@@ -228,14 +250,14 @@ export default function ClientIdentifierGenerator() {
 
     // set all fields to touched
     Object.keys(formValues).forEach((field) =>
-      form.setFieldTouched(field, true)
+      formik.setFieldTouched(field, true)
```

Comment:
why the rename?

Reply:
The formik references call the object `formik` as well so I figured, it'd be better to stick to that convention.